Yotpo/Yotpo/Block/Config.php

```diff
@@ -10,31 +10,53 @@ class Config
     const YOTPO_BOTTOMLINE_ENABLED = 'yotpo/settings/bottomline_enabled';
 
     public function __construct(
-        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
+        \Magento\Config\Model\Resource\Config $resourceConfig
     ) {        
         $this->_scopeConfig = $scopeConfig;
+        $this->_resourceConfig = $resourceConfig;
     }
 
     public function getAppKey()
     {   
-        return $this->_scopeConfig->getValue(self::YOTPO_APP_KEY, ScopeInterface::SCOPE_STORE);     
+        return $this->_scopeConfig->getValue(self::YOTPO_APP_KEY, ScopeInterface::SCOPE_STORES, 1);     
```

Comment:
@vladshub 
getValue expects 3 params, we added the last one for consistency (Alon's advice)